articles/iot-edge/how-to-retrieve-iot-edge-logs.md

```diff
@@ -49,6 +49,8 @@ The [Logger class in IoT Edge](https://github.com/Azure/iotedge/blob/master/edge
 Use the **GetModuleLogs** direct method to retrieve the logs of an IoT Edge module.
 
 >[!TIP]
+>Use the `since` and `until` filter options to limit the amount of logs retrieved. Calling this direct method without bounds will retrieve all the logs which may be large, time consuming or costly.
```

Comment:
```suggestion
>Use the `since` and `until` filter options to limit the range of logs retrieved. Calling this direct method without bounds retrieves all the logs which may be large, time consuming, or costly.
```